drivers/prune_tapdisk_logs

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+find /var/log/blktap/ -type f -mtime +30 -delete
+
+# Reopen the tlogs for any long running tapdisk processes
+/usr/bin/pkill -HUP "^tapdisk$" || true
```

Comment:
I guess it works in this case because it is the last character in the string but it would be better to use single-quotes to avoid `$` being interpolated by the shell.